extensions/wikia/Wall/WallExternalController.class.php

```diff
@@ -7,6 +7,8 @@
  */
 
 class WallExternalController extends WikiaController {
+	use \Wikia\Logger\Loggable;
```

Comment:
Yah, you can see it here:
https://github.com/Wikia/app/pull/8853/files#diff-8546689cd7c7a5fca56d659448f6c2e2R601